CenterForOpenScience/ember-osf-web
# [ENG-394] Add editable field wrapper component

## Purpose

Make editable field behavior consistent
Add editable field wrapper component

## Summary of Changes
 
## Side Effects

## Feature Flags

## QA Notes

Please make sure:
- admin can add/remove affiliated institutions.
- tags widget still works.
- saving edits works.
- discarding edits works.

## Ticket

[[ENG-394]](https://openscience.atlassian.net/browse/ENG-394)

# Reviewer Checklist
